lib/index.js

```diff
@@ -1,2 +1,4 @@
-export * from 'htm/preact';
+export { html, render } from 'htm/preact';
+
+export * from 'preact';
```

Comment:
Can we still ensure that `render` wasn't exported twice here (cf. https://github.com/bpmn-io/diagram-js-ui/pull/10/files)?

Reply:
Yes. Thanks for highlight, to not necrobump an old bug.